src/App.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import IHelloProps from 'interfaces/IAppProps'
+
+export default class App extends React.Component<IHelloProps, {}>{
```

Comment:
Why you changed this to class component?

Reply:
Actually, I just thought that in future we need an App component with the state, so change it here.

Reply:
If needed in future we can change it later.

Reply:
Okay, thanks.